test/dsl_test.go

```diff
@@ -232,7 +265,7 @@ func (o *opt) failf(format string, objs ...interface{}) {
 
 func (o *opt) expectSuccess(reason string, err error) {
 	if err != nil {
-		o.t.Fatalf("Error: %s: %v", reason, err)
+		o.t.Errorf("Error: %s: %v", reason, err)
```

Comment:
This was because calling `Fatalf` in test in a non-main routine (is an issue)[https://golang.org/pkg/testing/#T]. For me it just hang after the error. `Errorf` doesn't exit right away, but still fails the test and is usable in a different goroutine.